heron/spi/src/java/com/twitter/heron/spi/statemgr/SchedulerStateManagerAdaptor.java

```diff
@@ -209,6 +209,68 @@ public Boolean deleteSchedulerLocation(String topologyName) {
   }
 
   /**
+   * Checks to see if the execution state exists
+   *
+   * @return a boolean indicating whether the execution state exists or not
+   */
+  public boolean doesExecutionStateExist(String topologyName) {
+    return awaitResult(delegate.executionStateExists(topologyName));
+  }
+
+  /**
+   * Clean all states of a heron topology.
+   * TMasterLocation, PackingPlan, PhysicalPlan, SchedulerLocation, ExecutionState, and Topology
+   */
+  public Boolean cleanState(String topologyName) {
```

Comment:
The meaning of the return Boolean in ambiguous. Please define it in the javadocs.

Also most of the javadocs within this method are previously copy/pasted and invalid and could even be removed. I'd say we remove them all since they don't seem helpful.